survivalstan/utils.py

```diff
@@ -19,7 +19,7 @@ def _summarize_survival(df, time_col, event_col, evaluate_at=None):
     table = survival_table_from_events(df[time_col], df[event_col])
     table.reset_index(inplace=True)
     ## normalize survival as fraction of initial_n
-    table['initial_n'] = table.loc[table['event_at'] == 0.0,'at_risk'][0]
+    table['initial_n'] = max(table.at_risk)
```

Comment:
So the survival_table_from_events always starts at T = 0, so I could get the max number of subjects by looking at the value of `at_risk` at time `event_at == 0`. 

But, when I'm passing in `pp_data`, I'm not necessarily starting at time = 0. In fact, the use case for this was a scenario when I was starting at t = 90 going forward, so I needed to get the max number of subjects in a way that did not start at a particular time. It's cleaner to use the max of this value since that doesn't depend on the data structure.